ui/src/components/VolumeListTable.js

```diff
@@ -282,6 +282,13 @@ function Table({ columns, data, nodeName, rowClicked, volumeName, theme }) {
                         <div>{intl.translate('unknown')}</div>
                       </Cell>
                     );
+                  } else if (cell.column.Header === 'Name') {
+                    // add data-cy for delete volume E2E test
+                    return (
+                      <Cell {...cellProps} data-cy="table_cell_volume_name">
```

Comment:
IIUC, you're using this to count the number of rows. So, why not count `TableRow`s instead of `Cell`s?